heron/schedulers/src/java/com/twitter/heron/scheduler/yarn/HeronExecutorTask.java

```diff
@@ -133,6 +133,7 @@ public HeronExecutorTask(final REEFFileNames fileNames,
         env,
         topologyJar,
         topologyDefFile,
+        false,
```

Comment:
@ashvina Fantastic! We can merge #1208 first and make change on this PR accordingly.